### PR TITLE
Start the Django Gyst project

### LINKED ISSUE #1: Start the django project
Start a Django project using Django 2.2 and add initial files

AC: 
-Django version is 2.2
-has a requirements.txt file
-has a .gitignore file
-has a readme file

### PR DESCRIPTION
**Pull request by CrisKP**
This is the initial setup for the project. 
Closes #1

### CODE REVIEW

File: gyst/settings.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'i2$!s2p5fccdtmp4y$6r8d!=v*2f@)5fqyp0b4affqif9k-1xz'
```
Comment (HelloMelanieC) [inline gyst/settings.py]: We'll want to deal with removing this using a .env probably.

File: gyst/settings.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```
Comment (HelloMelanieC) [inline gyst/settings.py]: We'll want to do the same with this setting

File: gyst/settings.py (hunk 1 above)
Comment (CrisKP) [reply gyst/settings.py]: Yes, good catch. We'll fix it in issue #3.

File: gyst/settings.py (hunk 2 above)
Comment (CrisKP) [reply gyst/settings.py]: Agreed, will address in #3 too.

**Pull request merged**